node-operators/networks/collator.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Parachain Node
-description: How to run a Parachain node for the Moonbeam Network
+title: Collator
```

Comment:
"Collator" refers to the nodes that author blocks. It sounds like this will be the docs for a full node, so the old title, or maybe "Parachain Full Node" would be better.

Reply:
Forgot to hide this file. This was just to have as an internal reference that we will need a separate collator page because the one we will release is for a full node only.

We talked about calling them:
Full node
Collator node

Do you agree with this naming convention @JoshOrndorff ?